tests/beman/execution26/exec-sync-wait.test.cpp

```diff
@@ -223,6 +223,15 @@ auto test_sync_wait() -> void {
         // NOLINTEND(cert-dcl03-c,hicpp-static-assert,misc-static-assert)
     }
 }
+
+auto test_provides_scheduler() -> void {
```

Comment:
My tests aren't particular smart: they don't get picked up automatically, i.e., they still need to be listed in the "main" (`TEST(...)`) to actually get called. I _think_ these look like compile-time tests in which case they could probably use `static_assert`, e.g.:

     static_assert(requires{test_std::sync_wait(test_std::then(test_std::read_env(test_std::get_scheduler()), [](auto&&) noexcept {})); });

Reply:
It should be a runtime test, but its something I committed really quick just before lunch without checking whether it works. I just thought about, "oh, I'm fixing something. There should be a test."

Reply:
i have applied the relevant change and also fixed a problem discovered by adding that tests.